Modules/Keychain/README.md

```diff
@@ -0,0 +1,10 @@
+#  Keychain
+
+The keychain module provides an easy to use API agnostic to the device keychain and the iOS `Security` framework.
+
+```swift
+struct MyDataRepository {
+    @Secured(key: "<#keychain value identifier#>")
```

Comment:
Maybe we could add an example for the full usage, together with passing the keychain instance